db/seeds.rb

```diff
@@ -1,7 +1,18 @@
+# frozen_string_literal: true
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+puts "Seeding ..."
+
+Greeting.create(message: 'Hello')
+Greeting.create(message: 'Hi')
+Greeting.create(message: 'Greetings')
+Greeting.create(message: 'Niaje')
+Greeting.create(message: 'Howdy')
```

Comment:
* I get this error when i try to run `rails db:seed`  kindly fix it 🙏 as it doesn't display the greetings HINT 💭 i think you  can change the message to uppercase as your controller and database expect a `Message` column and params with a capital letter 👍 
![image](https://user-images.githubusercontent.com/52497246/140931531-67d5a8c3-d077-4c8d-a982-527d4bd24db6.png)

Reply:
@Davidosky007, Thank you for the hint.